### PR TITLE
Feature/sam k

### PR DESCRIPTION
**Pull request by rmudambi**
### Description

- *Category*: implementation
- *JIRA issue*: [MIC-2869](https://jira.ihme.washington.edu/browse/MIC-2869)
- *Research reference*: [PR](https://github.com/ihmeuw/vivarium_research/pull/765)

Adds a new scenario branch for an alternative sam-k value
Adds 6 months of additional burn-in

### Verification and Testing
Ran simulation

### CODE REVIEW

**Review by albrja — APPROVED** [review]

File: src/vivarium_ciff_sam/constants/data_values.py [hunk 1]
```diff
@@ -61,7 +61,12 @@ class __Wasting(NamedTuple):
     MAM_TX_ALTERNATIVE_EFFICACY: float = 0.75
 
     # Incidence correction factor (total exit rate)
-    SAM_K: float = ('sam_incidence_correction', get_lognorm_from_quantiles(median=6.7, lower=5.3, upper=8.4))
+    SAM_K: Tuple = (
+        'sam_incidence_correction', get_lognorm_from_quantiles(median=6.7, lower=5.3, upper=8.4)
+    )
+    ALTERNATIVE_SAM_K: Tuple = (
```
Comment (beatrixkh) [inline src/vivarium_ciff_sam/constants/data_values.py:67]: the research documentation linked mentioned bumping back the start of simulation run by 6 months to get a burn in period. is there a research reference for the "Adds a new scenario branch for an alternative sam-k value"?

File: src/vivarium_ciff_sam/constants/scenarios.py [hunk 2]
```diff
@@ -20,19 +22,40 @@ def __init__(
...
+        for scenario in self:
+            if scenario.name == item:
+                return scenario
+        raise KeyError(item)
+
+
+INTERVENTION_SCENARIOS = __InterventionScenarios()
+
+
+class SamKScenario:
```
Comment (beatrixkh) [inline src/vivarium_ciff_sam/constants/scenarios.py:43]: what is this / why is it named sam k?

**Review by beatrixkh — APPROVED** [review]

File: src/vivarium_ciff_sam/constants/data_values.py (hunk 1 above)
Comment (rmudambi) [reply src/vivarium_ciff_sam/constants/data_values.py:67]: I'm not sure if there is. It was mentioned in slack and in meetings though.

File: src/vivarium_ciff_sam/constants/scenarios.py (hunk 2 above)
Comment (rmudambi) [reply src/vivarium_ciff_sam/constants/scenarios.py:43]: This is governing the new scenario branch for sam k (the k is referring to a k-factor that governs differential remissions due to treatment)/

**Pull request merged**